src/Events/AliasNotFoundEvent.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+namespace Consolidation\SiteAlias\Events;
+
+use Consolidation\SiteAlias\SiteAlias;
+use Symfony\Contracts\EventDispatcher\Event;
+
+class AliasNotFoundEvent extends Event
```

Comment:
Naming things: I'd prefer to see this called something like AliasDiscoveryEvent, as AliasNotFoundEvent sounds like error reporting rather than a second-tier searching event.